cloudify_hostpool/storage/sqlite.py

```diff
@@ -155,7 +158,12 @@ def _create_table(self):
         with self.connect() as cursor:
             sql = 'CREATE TABLE IF NOT EXISTS {0} {1}'.format(
                 self.TABLE_NAME, self._schema.create())
-            cursor.execute(sql)
+            try:
+                cursor.execute(sql)
+            except sqlite3.OperationalError:
+                self._table_created = False
```

Comment:
what if one of the workers executes this code after the table was created? will the execution fails because the table already exists? or will it silently ignore and return?